### ISSUE TITLE
EPE= 1.384 on SceneFlow_cleanpass  using your pretrained model ??

### ISSUE THREAD

**Issue by wuzhongwulidong** [JiaRenChang/PSMNet#224]
Thanks for your great work!
Howerver,  I cant only get EPE=1.384 on SceneFlow_cleanpass  using your pretrained model ( disparity is multiplied by 1.17 already and align_corners=True)!
So, How can we reproduce the result EPE=1.09 in your paper?  I read through other issues, but it seems that this is an old issue that has not been solved yet. Can you release your test code on SceneFlow that can reproduce the result EPE=1.09. Thanks a lot.
Here is my result using your pretrained model:
epe: 1.384      d1: 0.0444      thres1: 0.1542

**Comment by Amaris286** [comment]
Yes, I tried. But I also got EPE=1.391. So do you solve this question?

**Comment by wuzhongwulidong** [comment]
I believe it‘s the same problem. No one can solve it except the author. Waiting for a reply...

**Comment by JiaRenChang** [comment]
Hi all,
    I release a newly trained model using torch 1.8.1, and you can see this update on README.
    [google drive](https://drive.google.com/file/d/1NDKrWHkwgMKtDwynXVU12emK3G5d5kkp/view?usp=sharing)
    Please set "align_corners=False" with this pretrained weight.
    This newly trained model should get EPE=0.974 according to my experiment recording.
    
    I guess the mismatched result may cause by the version of torch and cuda.
    I hope the new trained model can remove your doubts.

Jia-Ren

**Comment by Amaris286** [comment]
I tried and got a result 1.105. Thank you very much for your answer.
